test/test.tf

```diff
@@ -6,6 +6,31 @@ module "newrelic_monitoring" {
   newrelic_app_name                 = var.newrelic_app_name
   newrelic_fully_qualified_app_name = var.newrelic_fully_qualified_app_name
 
+  response_status_codes_alerts = [
+    {
+      name       = "${var.newrelic_app_name}: too many sustained 4xx errors"
+      nrql_value = "4%"
```

Comment:
`nrql_value` is not very descriptive. Let's use `status_code` or `status_code_pattern` (or something similar).